route/route.go

```diff
@@ -524,19 +530,29 @@ func (r *Router) processEvent(ev *types.Event, reqID interface{}) error {
 	// we know we're a span, but we need to check if we're in Stress Relief mode;
 	// if we are, then we want to make an immediate, deterministic trace decision
 	// and either drop or send the trace without even trying to cache or forward it.
+	isProbe := false
 	if r.Collector.Stressed() {
 		rate, keep, reason := r.Collector.GetStressedSampleRate(traceID)
 
 		r.Collector.ProcessSpanImmediately(span, keep, rate, reason)
-		return nil
+
+		if !keep {
+			return nil
+		}
+		// If the span was kept, we want to generate a probe that we'll forward
+		// to a peer IFF this span would have been forwarded.
```

Comment:
```suggestion
		// to a peer IF this span would have been forwarded.
```

Reply:
Heh, that was an abbreviation my math professors would use to mean "If and Only If" -- but I'll take this as criticism that not everyone knows that and accept your suggestion. :) Thanks for the review!

Reply:
I wondered if it was an abbreviation because it was in caps.